### DISCUSSION TITLE
Best way to share ui components between projects

### DISCUSSION THREAD

**Discussion by jrmoulton** [slint-ui/slint#823] [Q&A] [answered]
What's the best way to share ui compnents between different projects? So far I've just been copying the elements that I want to reuse but it would be nice if there was some way that I could just depend on a given ui library that was separate from the project and had it's own singular version control. But just sym linking another folder makes it so that the project can't just be cloned and built without downloading the library separately. What's the best way to share components?
Would sixtyfps benefit from having it's own package manager?

**Comment by ogoffart** [comment, marked as answer]
The way to do this is indeed to share the files. This can be done with git submodules and such.

We are open for suggestions on better way to do that. Having our own package manager is probably overkill. It would be better to try to use existing ones such as creates.io. I don't know if and how that would work though.
